ast/builtins.go

```diff
@@ -2159,6 +2166,76 @@ var HTTPSend = &Builtin{
 	),
 }
 
+/**
+ * GraphQL
+ */
+
+// GraphQLParse returns an object with results from parsing/validation.
+var GraphQLParse = &Builtin{
+	Name: "graphql.parse",
+	Decl: types.NewFunction(
+		types.Args(
+			types.S,
+			types.S,
+		),
+		types.NewArray([]types.Type{
+			types.NewObject(nil, types.NewDynamicProperty(types.A, types.A)),
+			types.NewObject(nil, types.NewDynamicProperty(types.A, types.A))}, nil),
+	),
+}
```

Comment:
These now should use named types, see https://github.com/open-policy-agent/opa/blob/e8deba62bd8a23e17cf79a080423eea4b1a5d2a8/ast/builtins.go#L692-L707 for a random example.

They don't need a category, it'll default to "graphql" because that's before the dot in their name.

Reply:
I have implemented named arguments on more recent versions of the PR, so I'm marking this as resolved. :smiley: